src/main/resources/templates/person/notifications/departments.html

```diff
@@ -78,6 +79,49 @@
             </label>
           </span>
         </div>
+        <div
+          class="list-selectable__item tw-flex tw-flex-col tw-gap-2 md:tw-flex-row md:tw-items-center md:tw-gap-4 xl:tw-gap-16"
+        >
+          <div class="tw-flex tw-flex-col tw-gap-2 md:tw-flex-row md:tw-items-center md:tw-gap-4 xl:tw-gap-16">
+            <div class="md:tw-w-64 lg:tw-w-72">
+              <span class="checkbox-switch tw-flex tw-items-center tw-gap-2 md:tw-gap-3">
+                <input
+                  type="checkbox"
+                  id="email-notification-department-restriction-input"
+                  name="restrictToDepartments"
+                  class="tw-mt-0.5 tw-shrink-0"
+                  th:checked="*{restrictToDepartments}"
+                  aria-describedby="info-email-notification-department-restriction-input"
+                  data-auto-submit
+                />
+                <span>
+                  <label
+                    for="email-notification-department-restriction-input"
+                    class="tw-m-0 tw-text-base tw-font-normal"
+                    th:text="#{person.notifications.departments.restriction.label}"
+                  >
+                    nur meine Abteilungen
```

Comment:
Für einen normalen _USER_ macht "nur meine Abteilungen" keinen Sinn. Dieser hat kein berechtigtes Interesse an der kompletten Organisation :-)

Sofern es Abteilungen gibt, ist das für ihn immer `true` und muss hier nicht angezeigt werden.

Reply:
und das bringt mich noch zu dem Punkt, dass diese Seite generell nur Sinn macht, wenn es Abteilungen gibt :-)

Reply:
Mittlerweile hat er das, da wir eine E-Mail-Notification hat, welche Benachrichtigungen über Abwesenheiten (nicht Krankmeldungen bzw. Urlaube) verschickt, sodass die Kollegen nicht in den Abwesenheitskalender schauen müssen sondern aktiv informiert werden.